sanic/testing.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, app):
                     session, method.lower())(url, *args, **kwargs) as response:
                 try:
                     response.text = await response.text()
+                    response.json = await response.json()
```

Comment:
This will break any code that uses `await response.json()` previously. It's a breaking change.